_sass/custom/custom.scss

```diff
@@ -48,11 +48,12 @@ code {
 }
 
 .site-nav {
-  padding-top: 1rem;
+  padding-top: 0;
   position: sticky;
-  top: 1rem;
+  top: 0;
   max-height: calc(100vh - 2rem);
```

Comment:
Do we need `- 2rem` now that the `top` and `padding-top` are `0`?

Reply:
We don't, thank you for catching this. Updated.